### DISCUSSION TITLE
Widows 10 Home 64

### DISCUSSION THREAD

**Discussion by mrBAHO2** [dbeaver/dbeaver#17195] [Q&A]
Hi Team,
Is DBeaver Community 22.1.3 compatible with Windows 10 **Home** Edition? I have some problems with connection.
![image](https://user-images.githubusercontent.com/110195996/181616979-2bfebc53-5747-4d95-acc1-78ef62a7b7a5.png)
Thanks!

**Comment by LonwoLonwo** [comment]
Hello @mrBAHO2 

Yes, DBeaver Community 22.1.3 is compatible with Windows 10 Home Edition.

Your issue looks like a PG driver/server issue. What PG driver version do you use? You can check it in "Настройки соединения"->"Библиотеки"

**Reply by LonwoLonwo** [reply]
https://github.com/pgjdbc/pgjdbc/blob/master/pgjdbc/src/main/java/org/postgresql/Driver.java#L252

**Reply by mrBAHO2** [reply]
Hi @LonwoLonwo, thanks for the reply! Please check the screenshot: 
![image](https://user-images.githubusercontent.com/110195996/182202345-1da7d7ca-46b0-4fe6-87e7-44a716b01bdf.png)

**Reply by LonwoLonwo** [reply]
Ok, this is a usual driver. The issue is not in the version.
Probably for this issue, you should contact your DBA.

**Reply by mrBAHO2** [reply]
@LonwoLonwo,
Thank you!
